packages/hd/src/keystore.ts

```diff
@@ -234,11 +239,15 @@ export default class Keystore {
 
   derivedKey(password: string): Buffer {
     const { kdfparams } = this.crypto;
-    return crypto.scryptSync(
-      password,
-      Buffer.from(kdfparams.salt, "hex"),
-      kdfparams.dklen,
-      Keystore.scryptOptions(kdfparams)
```

Comment:
this is a breaking change here, we need to be compatible with the previous `kdfparams`

Reply:
The method signature stays the same: `derivedKey: (password: string) => Buffer`

Reply:
I think I misunderstood, this is not a breaking change, just ignore this comment